src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -3687,7 +3687,22 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
             && symIdToTechn(sid) == ""
             && !a->isOrnament() && !a->isTapping()
             && !isLaissezVibrer(sid)) {
-            LOGD("unknown chord attribute %d %s", static_cast<int>(sid), muPrintable(a->translatedTypeUserName()));
+            String otherArtic = u"other-articulation";
+            otherArtic += color2xml(a);
+            otherArtic += ExportMusicXml::positioningAttributes(a);
+            if (a->anchor() != ArticulationAnchor::AUTO) {
+                if (a->anchor() == ArticulationAnchor::TOP) {
+                    otherArtic += u" placement=\"above\"";
+                } else {
+                    otherArtic += u" placement=\"below\"";
+                }
+            }
+            notations.tag(m_xml, a);
+            articulations.tag(m_xml);
+            AsciiStringView noteheadName = SymNames::nameForSymId(sid);
+            otherArtic += String(u" smufl=\"%1\"").arg(String::fromAscii(noteheadName.ascii()));
+            m_xml.tagRaw(otherArtic);
+            m_xml.endElement();
```

Comment:
I'm not super familiar with the MusicXML export code, but when comparing this to line 3536, I wonder why there is an `endElement` here and not there.

Reply:
Good question. I removed it.